src/nova_router.erl

```diff
@@ -525,6 +534,7 @@ print(#node{key = '__ROOT__', children = Children}) ->
     print(Children, 0).
 
 print([], _Level) -> ok;
+print([ok], _Level) -> ok;
```

Comment:
This we can remove after we have solved insert. But it kind of break now when we can have [ok] as child.